test/routes/report/report.test.js

```diff
@@ -471,11 +471,13 @@ describe('/reports/{REPORTID}', () => {
 
     test('/ - key variant - 200 Success', async () => {
       const res = await request
-        .get(`/api/reports?keyVariant=${KEYVARIANT}`)
+        .get(`/api/reports?keyVariant=${KEYVARIANT}&&matchingThreshold=1`)
         .auth(username, password)
         .type('json')
         .expect(HTTP_STATUS.OK);
 
+      console.log(res.body);
```

Comment:
Leftover debug code, could you remove it? After this it should be good to go :)

Reply:
thank you Lara let me remove it!